### DISCUSSION TITLE
自动运行“内存优化”

### DISCUSSION THREAD

**Discussion by ANcRyel** [Meloong-Git/PCL#2414] [讨论]
能在内存不够时自动运行“内存优化”吗，总是玩着玩着崩掉

**Comment by iPodtouch6557** [comment]
这个可以通过加`--memory`参数实现，当然不会或懒得搞的话，也可以看加速球快跑满了直接点内存优化按钮执行
现在也有在制作静默执行内存优化的功能，但是还没做完，你先别急.png

**Comment by XiaoLLiu617** [comment]
目前来说，内存优化对设备为机械硬盘的用户说并不友好（is me)J建议参数法解决
